modelci/app/v1/endpoints/model.py

```diff
@@ -52,6 +41,27 @@ def get_model(*, id: str):  # noqa
     return JSONResponse(content=content)
 
 
+@router.patch('/{id}', response_model=MLModel)
+def update(id: str, model: ModelUpdate):
+    prev_model = get_by_id(id)
+    if not prev_model:
+        raise HTTPException(
+            status_code=404,
+            detail=f'Model ID {id} does not exist. You may change the ID',
+        )
+    return update_model(id, model, prev_model)
```

Comment:
And since it has `id`, may be this function do not need an `prev_model` as input.